tests/foreman/ui/test_ldap_authentication.py

```diff
@@ -1406,3 +1417,66 @@ def test_verify_attribute_of_users_are_updated(session, ldap_data, ldap_tear_dow
         assert ldap_data['ldap_user_name'] in user_values['user']['firstname']
         assert ldap_data['ldap_user_name'] in user_values['user']['lastname']
         assert ldap_data['ldap_user_name'] in user_values['user']['mail']
+
+
+@pytest.mark.skip_if_open("BZ:1812688")
+@tier2
+def test_userlist_with_external_admin(session, auth_source_ipa, ldap_tear_down, groups_teardown):
+    """All the users should be displayed (internal and external) to all LDAP admin's either
```

Comment:
`All the users (internal and external) should be displayed to all LDAP admins (internal and external).`